public/html/projects/templates/add.pug

```diff
@@ -39,6 +45,11 @@
 			.col-sm-6
 				select.form-control(ng-model="tpl.environment_id" ng-options="env.id as env.name disable when env.removed for env in environment")
 					option(value="") -- Select Task Environment --
+		.form-group
+			label.control-label.col-sm-4 Vault Password
```

Comment:
This currently does not seem to filter on the vault password key type properly and shows all keys